src/components/searchBar/index.tsx

```diff
@@ -1,23 +1,86 @@
 import React, { useCallback, useState } from "react";
 
+interface SearchResultListProps {
+  id: number;
+  name: string;
+}
+
 export default function SearchBar() {
   const [value, setValue] = useState("");
+  const [searchResultList, setSearchResultList] = useState<SearchResultListProps[]>([]);
```

Comment:
Naming suggestions:
```suggestion
  const [searchResults, setSearchResults] = useState<SearchResult[]>([]);
```
or
```suggestion
  const [ingredients, setIngredients] = useState<Ingredient[]>([]);
```